tests/detabler/detabler.js

```diff
@@ -0,0 +1,187 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: contextmenu,detabler */
+
+(function () {
+	'use strict';
+	bender.editor = {
+		config: {
+			enterMode: CKEDITOR.ENTER_P
+		}
+	};
+	bender.test({
+		assertHtml: function (expected, actual, msg) {
+			assert.areEqual(bender.tools.fixHtml(expected), bender.tools.fixHtml(actual), msg);
+		},
+		setUp: function () {
+			this.command = this.editorBot.editor.getCommand('detable');
+		},
+		'it does not do anything outside of tables': function () {
+			var editor = this.editorBot.editor,
+				startHtml,
+				endHtml;
+
+			startHtml = '<p>foobar^</p>' +
+				'<p>baz</p>' +
+				'<table>' +
+					'<tbody>' +
+						'<tr><td>leave me alone</td></tr>' +
+					'</tbody>' +
+				'</table>';
+
+			endHtml = '<p>foobar</p>' +
+				'<p>baz</p>' +
+				'<table>' +
+					'<tbody>' +
+						'<tr><td>leave me alone</td></tr>' +
+					'</tbody>' +
+				'</table>';
```

Comment:
So the difference here is the cursor position? So, after toggling the context menu, the cursor position is gone?